packages/zoe/src/zoeService/zoe.js

```diff
@@ -15,27 +15,27 @@ import '@agoric/vats/exported.js';
 import '../internal-types.js';
 
 import { E } from '@endo/eventual-send';
-import {
-  makeScalarBigMapStore,
-  vivifyFarClassKit,
-  provide,
-} from '@agoric/vat-data';
+import { Far } from '@endo/marshal';
+import { makeScalarBigMapStore, vivifyFarInstance } from '@agoric/vat-data';
 
 import { makeZoeStorageManager } from './zoeStorageManager.js';
 import { makeStartInstance } from './startInstance.js';
 import { makeOfferMethod } from './offer/offer.js';
 import { makeInvitationQueryFns } from './invitationQueries.js';
-import { getZcfBundleCap, setupCreateZCFVat } from './createZCFVat.js';
+import { getZcfBundleCap } from './createZCFVat.js';
 import { defaultFeeIssuerConfig, vivifyFeeMint } from './feeMint.js';
-import { ZoeServiceIKit } from '../typeGuards.js';
+import { ZoeServiceI } from '../typeGuards.js';
 
 /** @typedef {import('@agoric/vat-data').Baggage} Baggage */
 
+const { Fail } = assert;
+
 /**
  * Create an instance of Zoe.
  *
- * @param {VatAdminSvc} vatAdminSvc - The vatAdmin Service, which carries the power
- * to create a new vat.
+ * @param {Promise<VatAdminSvc> | VatAdminSvc} [vatAdminSvcP] - The vatAdmin Service, which carries the
```

Comment:
I think this type can be simplified.
```suggestion
 * @param {ERef<VatAdminSvc>} [vatAdminSvcP] - The vatAdmin Service, which carries the
```

Reply:
Good catch.